docs/APISocketOptions.md

```diff
@@ -373,10 +374,20 @@ seconds). It is therefore strongly recommended that you only set this flag to
 FALSE on the listener when you are able to ensure that it is also set to FALSE
 on the caller side.
 
-
 [Return to list](#list-of-options)
 
+#### SRTO_ESTINPUTBW
+
+| Option Name         | Since | Binding | Type      | Units   | Default    | Range    | Dir |Entity |
+| :------------------ | :---: | :-----: | :-------: | :-----: | :--------: | :------: |:---:|:-----:|
+| `SRTO_ESTINPUTBW`   | 1.5.0 |         | `int64_t` | B/s     |            |          | R   | S     |
+
+- Returns estimated input bitrate in Bytes per second (B/s).
 
+- Input bitrate is only estimated when the corresponding mode is selected
```

Comment:
This info is better not added because the new option sets may require changes here anyway.

That's also not exactly true, at least not for the incoming OUTPACEMODE option. In the mode where it should use the maximum rate between the estimated one and set by INPUTBW, the estimated rate is still being done although INPUTBW option is set.

At best you may suggest that it doesn't make sense to read this option when MAXBW is set to -1 or > 0.